static/js/client.js

```diff
@@ -135,14 +135,21 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
     xScale = d3.time.scale()
       .domain(d3.extent(data, function (d) { return d.date; }));
 
-    yScale = d3.scale.log()
-      .domain([scaleBg(30), scaleBg(510)]);
+    if(app.defaults.bgScale == 'lin'){
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/768573671/issues/source?bid=2452379&fileBranchId=2457568#l138)